doc/releases/release-notes-3.7.rst

```diff
@@ -411,20 +441,41 @@ Drivers and Sensors
   * Added the c:func:`flash_fill` utility function which allows to write
     a single value across a provided range in a selected device.
   * Added support for RRAM on nrf54l15 devices.
+  * Added support of non busy wait polling on STM32 OSPI driver.
+  * Added support for STM32 XSPI external nor flash driver (:dtcompatible:`st,stm32-xspi-nor`).
+  * Added support for XIP on external nor flash on STM32 OSPI, QSPI and XSPI driver.
+  * STM32 OSPI driver: clk, dqs, ncs ports can now be configured by device tree configurable,
+    see :dtcompatible:`st,stm32-ospi`.
 
 * GNSS
 
 * GPIO
 
   * Added support for Ambiq Apollo3 series.
   * Added Broadcom Set-top box(brcmstb) SoC GPIO driver.
+  * Added c:macro:`STM32_GPIO_WKUP` flag which allows to configure a pin as wakeup source from
+    Power Off state on STM32 family.
```

Comment:
```suggestion
  * Added c:macro:`STM32_GPIO_WKUP` flag which allows to configure specific pins as wakeup source from
    Power Off state on STM32 L4, U5, WB, & WL SoC series.
```

Reply:
Used `Added c:macro:`STM32_GPIO_WKUP` flag which allows to configure specific pins as wakeup source from
    Power Off state on STM32 series supporting :kconfig:option:`CONFIG_POWEROFF`.`

Reply:
C0x supports Poweroff but does not support GPIO Wakeup pins yet.